channels/it.m3u

```diff
@@ -25,23 +25,23 @@ http://live.sloode.com:1935/canale21/live2/playlist.m3u8
dtl3q+GbsMVlra9LnrjkqMmbnIhVCXQjVXEpN5Sh9GVTXruAMxVVakUYkr9YdOIN4ADreg1XglDpFcTgQx39wG5djX71ijuxn3sKue6oJJC0+TxrtXoYfnn8K71jG0z9UovqbtB6LUYJuOfRLPdCanH97w2YJQl/Y6JelCqPhItnu8FFPJK1NRwL+JTksX2QJVIGANCP7lu46gHquWl07oXgUOtkop10YNxc96Bm7LSOXituSwpNjnqQe/3ATgEV7vI3g1AvhQDjIu43PMHGtaP935uukKLYMdZg1rBGbBjtsttg0oz1/G22MbpntTyx1h21q/7zJtbhSx4j9GVLHlRZbj5Avb80bXULWoKPguDXUMVqr7axX2eNdD1uJsMi5mR0s3yGeYTF+tbVgQTi+WBt1p9ZEj2TDVL/gXWgrR/9kU5O8VYCpesaCDqqMiFI4xTr3SbHZxvCG0LvFA3BCsR15HBu+5vlNSFplfVPf8Ix/b8/Z+radX1PH4rKwDcWs9N+/UuzsqgHmzTWAeCXUpV/0KXsUKk91pk7JrGcIpbEtwvgjkyjaIQdJCp5jeAGTE2manOewZTkH+S4a+LfiL9F8Ng01NvtctguNmqnq9hpbBWeO2YUrL7Dt1+IkXb+8AoOTCoSvH2k6D2QUt8aq/Ku0AKHnFaSpBpvsI2bTCN5yWWLKFzD+N7hq0eCvrpd0/tciUpiFvdUjL+R1i1RtqpXBRED4RHF6U0EAAyeawDeeOZjnwBmsPH/t6U2iMwvJV7n0MBR1vIMB51g38C2eBTC2kLDVU6b83N88HGmijf1AeZU7tZiSxqMlEqjNXCsYZhlJvCRZGYVbGr0Xqy1Uf70HAsJs26HYPRxO3jNrrjKTIdVzxQw5IbfGY31T8COdD9DNyZ8ImaVwdYiqUT/agB+ncRsgGJATNynEkaerQnh3Wv6bQ202kLW53HHQqYERlW2m5W/Vo9cxKXs+rFZyqitzzNdBO3jdCwA0YXZLcbgTDfHXxihH8YZNI2LdWlIEa8rb9jLVNWIf31LXPRpRwzB0BjV8rOz912Qkd6f1BYy+yMzjeSJhMbp6PfzrqT9qxUVx+8ON9qgT99KWRAo+ahTjRsbTY9LXOgXA7O8NtrR3cfCIKuB1cUzaqQsgotiuRpQPt0K5p1tsfsD6w0p6pR8UvjncJgZbPJOUz3145e++U07kkGGneJZa3gWQ/FCVZyBRG2TvLc3/95/c5/9wU58iIwMT4DvOfXp1gsiA1b1oM9iS+SUxKulJAW/WDjIlPYH7SXtpLQe+3BpHu2PvSdUsUy5/k+uMEJZyH3eKW5MyVYjqiQgVpS8c6qUb5d9efdAFivzfNxT/xwr/pSwyVG8NH6F1sd112VgYt45jt2p9z0S6oaeZ633akG5bKd3jSuxnlQS3Baan7oiX40OpmOsNcw5xEtj3u3WX25K6BeIjtB7AV7PJ9/j92MsKfjNBPZKzuao3+M88OYaphJHwYuYQQ92UHhEu9xNwo2QO9rGrp+vWcBtpvBxuS7L8i1OzfeJ2R1fI/tWhpjcBgCT9MJ1o7pWK9WHFUr5nyvibnaYxiUT7pOzPo8mHzRYXu2PQXFxvIdW5hGWgpoQ8+UK4KtwUw7BzMjm7snS/smcFg1UPfj1JSGrzDrT7zf3tp5swXqAxhHBRNAvdb8/jiaDO4Q6BDbGU10tECxK4D+stQB5cvB8dzEfhZfETsBzx/7pgQ2FRKrHeEOCKkc9reRe6/W7dIeuAVuxG8WMMbJZWbFr+0UdbHrDXrt0zPfgus5aCZjC72z+7bSPgcwYUXVdkWUFtz9zQ5aa9jxQDQzfnA29O00eFI3WZ+i21a66Z+vIXHmwBrDRYjByImYCnY+c3lzYyBE9o131rofEF1/ujburkEZuMFsszqeZJFa5SyGGkivmDmwghbogln45mOhEovZtF1+Muq2RcdqXImHS6droWOpwLfIwB0+Edu+Ze8j5HRbSAGrbKXRGxT8ZpadI1RYaK8m8YVpce1oY/ths2yXKlFkWYaHLWQ8K4oq6ZJdbXjjs25sEoi2dqgLeoW6LxpVMvScKtc0lPlKgMKll7FLasxv1KXbZf8CRQXOB3YbFBs3gdW2f1JY2zm6ODjA/kD10dOjs7XCZNCPTQAZW/XKIhVm+tM4SIsFJdUE3Vkyu6LX9jeDlIjG6t28HQk/MQyxpaxdrb8QrgK2ahctg1f3fYV5y+VAzWBnQMROH3wYNtOWKsW9HVSvxU+wD9mXcJJuCpqCnf1gdJOxb6ECpyIHjhv5cvSP7iAeGnbCVbALgSzPeg/qOloIouwjFe9p2tt/OXbKluisFPfZtILTB8KC0u4zFnNnAMP0yCmZAj/VE/LTqNjB+Kg7mM7wROwURbRV07TFrGbbk7VRrWpXDIrrPbgTsGdadufwO+rpQ+I+EfoEWAicence4afbRqXpW8P6UaYwedCeLs1FbzooFtbOLsZq2gGtwmjckRP4manmpl8vKmi1fTuEPgYtlc2i7Ox1z6tpg8/dYS7GTE0rXcpXSkF1q5zPp9Ppbd/3y07Jzm8aJjNBZuJeQbH6UCMMRqF6B/P3K+u/T0EKd8EVmwaSabe+W7bMJPSJxnay/zTum4KRXDRKRfQfo4eqYDwKUgXtcLPS8dcLYqOWFGjGP7yo9MVA3z/cCEoROrOIncRMV2iTpJE4RjEEfhJiL5xlOGRs2s4480UDFPcFwlkkvQcfTu1qpWJ5oikMuvyAlGgzh+zGPA/mTwUXyvU8Y7Oz1aq/HyBRfmKv6O8GxGacOaGk+5PM0+M8ePFpsGcAqUWfQ9AqpmlZtl1uGsdH7+NYlK6DUfM9aIbPQArt5izaNLEslFFqZo7WXw+90u7DGKplr7axfpR5VWzHQRwFm8Sk4raePzIoctKD8wcgxX/DNbDcvedbj+Xf8APB/wcZ4zHoAt/5jAAAAABJRU5ErkJggg==" group-title="",Carina TV (480p)
```

Comment:
Please do not use base64 encoded images here.

Reply:
Removed